clickhouse_search/models.py

```diff
@@ -176,6 +424,48 @@ class Meta:
         engine = EmbeddedRocksDB(0, f'{CLICKHOUSE_DATA_DIR}/GRCh38/SNV_INDEL/annotations', primary_key='key', flatten_nested=0)
 
 
+class Clinvar(models.ClickhouseModel):
+
+    PATHOGENICITY_CHOICES = list(enumerate([
+        'Pathogenic', 'Pathogenic/Likely_pathogenic', 'Pathogenic/Likely_pathogenic/Established_risk_allele',
+        'Pathogenic/Likely_pathogenic/Likely_risk_allele', 'Pathogenic/Likely_risk_allele', 'Likely_pathogenic', 'Likely_pathogenic/Likely_risk_allele',
+        'Established_risk_allele', 'Likely_risk_allele', 'Conflicting_classifications_of_pathogenicity',
+        'Uncertain_risk_allele', 'Uncertain_significance/Uncertain_risk_allele', 'Uncertain_significance',
+        'No_pathogenic_assertion', 'Likely_benign', 'Benign/Likely_benign', 'Benign'
+    ]))
+
+    key = ForeignKey('EntriesSnvIndel', db_column='key', related_name='clinvar_join', primary_key=True, on_delete=PROTECT)
```

Comment:
this field changed to be a `ForeignKey` on the entries table instead of a `OneToOneField` on the annotations table